src/mock-event-bus/index.test.ts

```diff
@@ -11,25 +11,26 @@ interface TestEventPayload2 {
     b: number;
 }
 
-type MockEvent1 = Event<TestEventPayload1>;
-type MockEvent2 = Event<TestEventPayload2>;
+type MockEventType1 = Event<TestEventPayload1>;
+type MockEventType2 = Event<TestEventPayload2>;
 
 describe('mock message queue', () => {
+    // describe('object lifetime', () => {
```

Comment:
Can this be removed?

Reply:
This was just commited to get the build working - this file is still half-baked :smile: